styles/shinigami-eyes/catppuccin.user.css

```diff
@@ -0,0 +1,143 @@
+/* ==UserStyle==
+@name Shinigami Eyes Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/shinigami-eyes
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/shinigami-eyes
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/shinigami-eyes/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Ashinigami-eyes
+@description Soothing pastel theme for Shinigami Eyes
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document
```

Comment:
Please add a comment detailing where this list is from.

Reply:
Yup :3

Reply:
https://github.com/catppuccin/userstyles/pull/1026